actors/builtin/codes.go

```diff
@@ -21,45 +21,41 @@ var (
 	CallerTypesSignable         []cid.Cid
 )
 
+var builtinActors map[cid.Cid]string
```

Comment:
Thanks. I later want to put more info in this map, like whether the actor is singleton, whether a "signer", etc.